resources/views/contacts/aboutUs.blade.php

```diff
@@ -74,20 +74,14 @@
                     </div>
 
                     <div class="row mx-0">
-                        {{-- <div class="bg-color-Rainbow" style="height: 140px;"></div> --}}
-                        <div id="map"></div>
-                        {{-- <gmp-map center="35.71015548706055,139.80955505371094" zoom="14" map-id="DEMO_MAP_ID">
-                        <gmp-advanced-marker position="35.71015548706055,139.80955505371094" title="My location"></gmp-advanced-marker>
-                        </gmp-map> --}}
+                        <div id="map" style="height: 140px;"></div>
                     </div>
                 </div>
             </div>
         </div>
         <div class="col-md-auto"></div>
     </div>
 </div>
-<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDYnarELUH_tcoAUGg6pLxho5piihVR_js&callback=initMap" async defer></script>
-{{-- <script async src="https://maps.googleapis.com/maps/api/js?key=&callback=console.debug&libraries=maps,marker&v=beta">
-</script> --}}
+<script src="http://maps.google.com/maps/api/js?key=AIzaSyDYnarELUH_tcoAUGg6pLxho5piihVR_js&language=en"></script>
```

Comment:
add the key in the env file

Reply:
let me know how I can add the key in the env file please

Reply:
@Mariee240 you can use env('VARIABLENAME'); and add variable in .env file